modules/ossm-accessing-bookinfo-application-using-istio-gateway-injection.adoc

```diff
@@ -0,0 +1,44 @@
+:_mod-docs-content-type: Procedure
+[id="ossm-accessing-bookinfo-application-using-istio-gateway-injection"]
+= Accessing the Bookinfo application by using Istio gateway injection
+:context: ossm-accessing-bookinfo-application-using-istio-gateway-injection
+
+Gateway injection uses the same mechanisms as {istio} sidecar injection to create a gateway from a `Deployment` resource that is paired with a `Service` resource. The `Service` resource can be made accessible from outside an {ocp-product-title} cluster.
+
+.Procedure
+
+. Create the `istio-ingressgateway` deployment and service by running the following command at the CLI:
++
+[source,terminal]
+----
+$ oc apply -n istio-system -f ingress-gateway.yaml
```

Comment:
Where does this yaml file exist?

Reply:
Good point, I mentioned it resides in the upstream repo and provided a link.